web/ui/react-app/src/App.tsx

```diff
@@ -25,6 +25,7 @@ const App: FC<AppProps> = ({ consolesLink }) => {
     '/config',
     '/rules',
     '/targets',
+    '/remote-writes',
```

Comment:
I think this should be singular, there are multiple queues, but it is the status for remote-write in general.